src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
```

Comment:
It is worth sticking to one style: either all variables through enam, or as a set of types.